### PR TITLE
Add Digibyte Support

### PR DESCRIPTION
**Pull request by esotericizm**

### CODE REVIEW

File: src/networks.js [hunk 1]
```diff
@@ -124,7 +124,20 @@ var networks = {
...
     estimateFee: estimateFee('zetacoin')
-  }
+  },
+  digibyte: {
+    magicPrefix: '\x18Digibyte Signed Message:\n',
+    bip32: {
+      public: 0x0488b21e,
+      private: 0x0488ade4
+    },
+    pubKeyHash: 0x1e,
```
Comment (dcousens) [inline src/networks.js]: same as `dogecoin`

File: src/networks.js [hunk 2]
```diff
@@ -124,7 +124,20 @@ var networks = {
...
+    scriptHash: 0x05,
```
Comment (dcousens) [inline src/networks.js]: same as `bitcoin`

File: src/networks.js [hunk 3]
```diff
@@ -124,7 +124,20 @@ var networks = {
     dustThreshold: 546, // https://github.com/zetacoin/zetacoin/blob/master/src/core.h#L159
     feePerKb: 10000, // https://github.com/zetacoin/zetacoin/blob/master/src/main.cpp#L54
     estimateFee: estimateFee('zetacoin')
-  }
+  },
+  digibyte: {
+    magicPrefix: '\x18Digibyte Signed Message:\n',
+    bip32: {
+      public: 0x0488b21e,
+      private: 0x0488ade4
```
Comment (dcousens) [inline src/networks.js]: both the same as `bitcoin`

File: src/networks.js [hunk 4]
```diff
@@ -124,7 +124,20 @@ var networks = {
...
+    },
+    pubKeyHash: 0x1e,
+    scriptHash: 0x05,
+    wif: 0x80,
```
Comment (dcousens) [inline src/networks.js]: same as `bitcoin`

**Comment by dcousens** [comment]
See https://github.com/bitcoinjs/bitcoinjs-lib/pull/338 for more information.

**Comment by weilu** [comment]
@esotericizm looks like you are trying to fork this library exclusively for digibyte.

**Pull request closed without merging**